### PR TITLE
Upgrade pnpm-lock using v9

### LINKED ISSUE #161: Upgrade pnpm-lock using v9
- [ ] Update pnpm version definition in `package.json`: https://github.com/mazipan/tanyaaja.in/blob/master/package.json#L81
- [ ] Install pnpm v9 in your local machine: https://pnpm.io/cli/self-update
- [ ] Remove old `pnpm-lock.yaml`
- [ ] Regenerate new `pnpm-lock.yaml` by running `pnpm i`

### PR DESCRIPTION
**Pull request by Piyush-Tilokani**
Closes #161 

## Description

Upgraded the `pnpm-lock.yaml` file using pnpm version 9.12.2 locally as specified in the issue

## Current Tasks

- [x] Update pnpm version definition in package.json: https://github.com/mazipan/tanyaaja.in/blob/master/package.json#L81
- [x] Install pnpm v9 in your local machine: https://pnpm.io/cli/self-update
- [x] Remove old pnpm-lock.yaml
- [x] Regenerate new pnpm-lock.yaml by running pnpm i

@mazipan please review and merge

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,6 +78,6 @@
   "packageManager": "pnpm@8.6.5",
   "engines": {
     "node": "20.x",
-    "pnpm": ">=8"
+    "pnpm": "9.x"
```
Comment (mazipan) [inline package.json]: Sample: https://vercel.com/docs/errors/error-list#pnpm-engine-unsupported

**Review by mazipan — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -78,6 +78,6 @@
   "packageManager": "pnpm@8.6.5",
```
Comment (mazipan) [inline package.json]: Update to `pnpm@9.12.2`

File: package.json [hunk 3]
```diff
@@ -78,6 +78,6 @@
...
   "engines": {
     "node": "20.x",
-    "pnpm": ">=8"
```
Comment (mazipan) [inline package.json:81]: Update to `"pnpm": "^9.12.2"

File: package.json (hunk 1 above)
Comment (Piyush-Tilokani) [reply package.json]: Do I need to specify the exact version that I used locally to update `pnpm-lock.yaml` ?

**Review by mazipan — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Piyush-Tilokani) [reply package.json:81]: Done in the latest commit

**Comment by Piyush-Tilokani** [comment]
@mazipan made the necessary changes in the latest commit
Please review and merge 
Do let me know if any changes are required

**Review by mazipan — APPROVED** [review]

**Pull request merged**